templates/checks.json

```diff
@@ -0,0 +1,11 @@
+{
+  "checks": { {% for check in checks %}
```

Comment:
That's what the `-`'s are for: http://jinja.pocoo.org/docs/2.9/templates/#whitespace-control